libgalois/include/katana/PagePool.h

```diff
@@ -61,6 +62,7 @@ typedef katana::CacheLineStorage<HeadPtr> HeadPtrStorage;
 template <typename _UNUSED = void>
 class PageAllocState {
   std::deque<std::atomic<int>> counts;
+  std::deque<std::atomic<int>> free_counts;
```

Comment:
If we aren't going to switch the style over for the entire class, I'd prefer we just stick to whatever naming convention we had before, `freeCounts`, `freeCount`, `freeCountAll`